samples: bluetooth: HID SCI support in peripheral HID samples (#28937)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

* :ref:`peripheral_hids_mouse` sample:

* Added:

  * Support for continuous HID report sending (one report notification for each connection interval) as well as statistics for continuous report sending.
  * A configuration with support for the HID Shorter Connection Intervals (SCI).

>     It is enabled by using the ``hid_sci`` suffix.

Reviewer: ```suggestion
    It is enabled with the ``hid_sci`` suffix.
```

* :ref:`peripheral_hids_mouse` sample:

  * Added support for continuous HID report sending (one report notification for each connection interval) as well as statistics for continuous report sending.

>   * Added configuration with support for the HID Shorter Connection Intervals (SCI).

Reviewer: ```suggestion
* Added:
 
  * Support for continuous HID report sending (one report notification for each connection interval) as well as statistics for continuous report sending.
  * A configuration with support for the HID Shorter Connection Intervals (SCI).
```
Should the configuration be mentioned explicitly (like a Kconfig option)?
